### PR TITLE
Release notes for Percona Server 5.6.36-82.1 updated to fix errors fo…

### PR DESCRIPTION
**Pull request by borys-belinsky**
…und during the review

### CODE REVIEW

**Review by percona-ysorokin — APPROVED** [review]
LGTM

File: doc/source/release-notes/Percona-Server-5.6.36-82.1.rst [hunk 1]
```diff
@@ -20,43 +20,83 @@ milestone at Launchpad
...
- When connecting to the Aurora cluster end point using SSL, SAN (Subject
- Alternative Name) certificates were ignored.  Bug fixed :bug:`1673656` (upstream
- :mysqlbug:`68052`).
+ |Percona FT| did not pass cmake flags on to snappy cmake. Bug fixed
+ :tdbbug:`41`.  The progress status for partitioned TokuDB table ALTERs was
+ misleading. Bug fixed :tdbbug:`42`.
 
- Killing a stored procedure which contained a query with a view resulted in a
- debug assertion failure. Bug fixed :bug:`1689736` (upstream :mysqlbug:`86260`).
+ When a client application connecting to the Aurora cluster end point using SSL
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.36-82.1.rst]: Previous suggestion not implemented? (no need to mention Aurora; Yura's OpenSSL version comment not added)

File: doc/source/release-notes/Percona-Server-5.6.36-82.1.rst [hunk 2]
```diff
@@ -20,43 +20,83 @@ milestone at Launchpad
...
 :bug:`1676740`,
 :bug:`1689989`,
 :bug:`1689998`,
 :bug:`1690012`,
 :bug:`1699788`, and
 :bug:`1684601` (upstream :mysqlbug:`86016`).
+
+.. _compatibility-matrix:
+
+Compatibility Matrix
```
Comment (laurynas-biveinis) [inline doc/source/release-notes/Percona-Server-5.6.36-82.1.rst:96]: If we do this, this has to go to some section in the main docs in addition to RNs (nobody will find it there after this release)

File: doc/source/upstream-bug-fixes.rst [hunk 3]
```diff
@@ -139,9 +139,9 @@
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`82980` - Multi-threaded slave leaks worker threads in case of thread create ...    |
 |:Launchpad bug: :bug:`1619622`                                                                               |
-|:Upstream state: Verified (checked on 2017-05-11)                                                            |
+|:Upstream state: Closed                                                                                      |
 |:Fix Released: :rn:`5.6.33-79.0`                                                                             |
-|:Upstream fix: N/A                                                                                           |
+|:Upstream fix: 5.6                                                                                           |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: 5.6.38

File: doc/source/upstream-bug-fixes.rst [hunk 4]
```diff
@@ -205,7 +205,7 @@
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`81714` - mysqldump get_view_structure does not free MYSQL_RES in one error path    |
 |:Launchpad bug: :bug:`1588845`                                                                               |
-|:Upstream state: Verified (checked on 2017-05-11)                                                            |
+|:Upstream state: Closed                                                                                      |
 |:Fix Released: :rn:`5.6.31-77.0`                                                                             |
 |:Upstream fix: N/A                                                                                           |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: 5.6.38

File: doc/source/upstream-bug-fixes.rst [hunk 5]
```diff
@@ -7,25 +7,25 @@
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`83814` - Add support for OpenSSL 1.1                                               |
 |:Launchpad bug: :bug:`1702903`                                                                               |
-|:Upstream state: Verified (checked on 2017-27-07)                                                            |
+|:Upstream state: Verified (checked on 2017-28-07)                                                            |
```
Comment (laurynas-biveinis) [inline doc/source/upstream-bug-fixes.rst]: YYYY-MM-DD

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Comment by laurynas-biveinis** [comment]
https://github.com/percona/percona-server/pull/1854/commits/364d507dd6a28d2eaee8e2b0c674fb245eb5fffb and https://github.com/percona/percona-server/pull/1854/commits/7e67de9efb8a89d568cabe1ae4b224c677fbd796 LGTM

File: doc/source/release-notes/Percona-Server-5.6.36-82.1.rst (hunk 2 above)
Comment (borys-belinsky) [reply doc/source/release-notes/Percona-Server-5.6.36-82.1.rst:96]: I have created a ticket to update the documentation: https://jira.percona.com/browse/DOC-572

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**